$OpenBSD$

fix src/lib/Evas_GL.h:385: error: nonnull argument with out-of-range operand number (argument 1, operand 2)

--- src/lib/Evas_GL.h.orig	Sun Oct 30 17:01:30 2011
+++ src/lib/Evas_GL.h	Tue Nov  1 18:11:47 2011
@@ -386,7 +386,7 @@ EAPI Evas_GL                 *evas_gl_new             
  *
  * @param evas_gl The given Evas_GL object.
  */
-EAPI void                     evas_gl_free               (Evas_GL *evas_gl) EINA_ARG_NONNULL(1, 2);;
+EAPI void                     evas_gl_free               (Evas_GL *evas_gl) EINA_ARG_NONNULL(1);
 
 /**
  * Creates and returns new Evas_GL_Surface object for GL Rendering.
